docs/resources/gaussdb_mysql_parameter_template.md

```diff
@@ -38,6 +65,31 @@ The following arguments are supported:
 
   Changing this parameter will create a new resource.
 
+  -> **NOTE:** It is mandatory when `datastore_engine` is separated.
+
+* `source_configuration_id` - (Optional, String, ForceNew) Specifies the source parameter template ID.
+
+  Changing this parameter will create a new resource.
+
+* `instance_id` - (Optional, String, ForceNew) Specifies the ID of the GaussDB MySQL instance.
+
+  Changing this parameter will create a new resource.
+
+* `instance_configuration_id` - (Optional, String, ForceNew) Specifies the parameter template ID of the GaussDB MySQL
+  instance.
+
+  Changing this parameter will create a new resource.
+
+  -> **NOTE:** It is mandatory when `instance_id` is separated.
+
+-> **NOTE:** 1. At most one of `datastore_engine`, `source_configuration_id` and `instance_id` can be separated.
+  <br>2. If `source_configuration_id` is separated, then the resource will replicate the parameter template separated
+  by `source_configuration_id`.
+  <br>3. If `instance_id` is separated, then the resource will replicate the parameter template of the GaussDB MySQL
+  instance separated by `instance_id`.
+  <br>4. If `source_configuration_id` and `instance_id` are both not separated, then a new parameter template will be
+  created directly.
```

Comment:
```suggestion
  <br>4. If `datastore_engine` is specified, then a new parameter template will be
  created directly.
```

Reply:
no, datastore_engine is not required, it can be empty, that is to say,  source_configuration_id, instance_id and datastore_engine can be null at the same time